### PR TITLE
Document mbstring.regex_retry_limit and mbstring.regex_stack_limit directives

### LINKED ISSUE #1823: Undocumented mbstring regex INI directives
The `mbstring.regex_retry_limit` and `mbstring.regex_stack_limit` are undocumented.

See:
 * `mbstring.regex_stack_limit`: https://github.com/php/php-src/commit/738016bd884a339009e1af371eaba0fee60bf23b (PHP 7.3.5)
 * `mbstring.regex_retry_limit`: https://github.com/php/php-src/commit/6623e7ac51f0bafe85e013f9aed5340199e80c10 (PHP 7.4.0)

### PR DESCRIPTION
**Pull request by kevincupp**
Resolves #1823

### CODE REVIEW

File: reference/mbstring/ini.xml [hunk 1]
```diff
@@ -235,6 +247,29 @@
...
    </varlistentry>
+   <varlistentry xml:id="ini.mbstring.regex-retry-limit">
+    <term>
+     <parameter>mbstring.regex_retry_limit</parameter>
+     <type>int</type>
+    </term>
+    <listitem>
+     <para>
+      Limits the amount of backtracking that may be performed during one mbregex
+      match.
```
Comment (cmb69) [inline reference/mbstring/ini.xml:258]: I think it is important to mention that:
```suggestion
      match.
      This setting only takes effect when linking against oniguruma >= 6.8.0.
```

**Review by cmb69 — COMMENTED** [review]
Thank you! Besides the issue mentioned below, this looks good to me.

File: reference/mbstring/ini.xml (hunk 1 above)
Comment (kevincupp) [reply reference/mbstring/ini.xml:258]: Good call! I added it below as a new paragraph, if that's what was intended by the line break

Comment (cmb69) [reply reference/mbstring/ini.xml:258]: Well, that wasn't my intention, but it's not a bad idea to have a separate paragraph. :)

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**